x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/types.ts

```diff
@@ -202,3 +203,8 @@ export interface FindRulesReferencedByExceptionsProps {
   lists: FindRulesReferencedByExceptionsListProp[];
   signal?: AbortSignal;
 }
+
+export enum RuleSourceTypesEnum {
+  MODIFIED = 'MODIFIED',
+  UNMODIFIED = 'UNMODIFIED',
```

Comment:
Nit: we use lowercase enum keys most of the time (when we don't need to map them on a string constant).